test_pytest_html.py

```diff
@@ -261,6 +255,31 @@ def pytest_runtest_makereport(item, call):
             href)
         assert link in html
 
+    def test_extra_text_separeted(self, testdir):
+        content = str(random.random())
+        testdir.makeconftest("""
+            import pytest
+            @pytest.mark.hookwrapper
+            def pytest_runtest_makereport(item, call):
+                outcome = yield
+                report = outcome.get_result()
+                if report.when == 'call':
+                    from pytest_html import extras
+                    report.extra = [extras.text('{0}')]
+        """.format(content))
+        testdir.makepyfile('def test_pass(): pass')
+        result, html = run(testdir)
+        assert result.ret == 0
+        hash_key = ('test_extra_text_separeted.py::'
```

Comment:
Can we get the name from `request` fixture? I recall mentioning this on another pull but perhaps we didn't do it there..